fleet_packages.json

```diff
@@ -40,6 +40,12 @@
     "name": "fleet_server",
     "version": "1.3.0"
   },
+  {
+    "name": "profiler_symbolizer",
+    "version": "8.8.0-preview",
+    "forceAlignStackVersion": true,
```

Comment:
no.

```
forceAlignStackVersion causes the bundled package's version to be updated to be exactly the same as Kibana's
```

Reply:
Ah ok - but then we need to change the package policy to reference the _Kibana version_ rather than a fixed version?
If this is happening: `the bundled package's version to be updated to be exactly the same as Kibana's`, doesit mean that in the package policy we'll need to install the package with 

```
  "package": {
    "name": "profiler_symbolizer",
    "version": "<KIBANA_VERSION>"
  },
```
?

I am asking because i don't understand why we are setting that to `true`, what are the implications?

Reply:
We are setting `forceAlignStackVersion` and `allowSyncToPrerelease` to `true` for the same reason as [apm-server does](https://github.com/elastic/apm-server/issues/10385).
Yes, once this bundling is happening, the package needs to be installed via kibana using the kibana version.